myning/research.yaml

```diff
@@ -37,17 +37,16 @@ species_discovery:
 species_rarity:
   name: Increase Companion Rarity
   descriptions:
-    - Increase the likelihood of recruiting more rare species
-    - Increase the likelihood of recruiting more rare species
-    - Increase the likelihood of recruiting more rare species
-    - Increase the likelihood of recruiting more rare species
-    - Increase the likelihood of recruiting more rare species
-    - Increase the likelihood of recruiting more rare species
-    - Increase the likelihood of recruiting more rare species
-    - Increase the likelihood of recruiting more rare species
-    - Increase the likelihood of recruiting more rare species
-    - Increase the likelihood of recruiting more rare species
-    - Increase the likelihood of recruiting more rare species
+    - Increase the likelihood of recruiting more rare species by 10%
+    - Increase the likelihood of recruiting more rare species by 20%
+    - Increase the likelihood of recruiting more rare species by 30%
+    - Increase the likelihood of recruiting more rare species by 40%
+    - Increase the likelihood of recruiting more rare species by 50%
+    - Increase the likelihood of recruiting more rare species by 60%
+    - Increase the likelihood of recruiting more rare species by 70%
+    - Increase the likelihood of recruiting more rare species by 80%
+    - Increase the likelihood of recruiting more rare species by 90%
+    - Increase the likelihood of recruiting more rare species by 100%
```

Comment:
This is not related to percentage at all. It needs to change back.

Reply:
What is it related to? There are 10, 20, etc. values but the descriptions don't reflect those value changes.

Reply:
Is it the common, uncommon, etc?

Reply:
I'm new to python, but I think this is the line that does the math.
```py
weight + RESEARCH["species_rarity"].player_value + (i * 3)
```

Reply:
It is related to the tier, but the increases are not neat percentages